README.md

```diff
@@ -91,10 +101,16 @@ large amounts of storage space.
 ```sql
 -- A policy using the REALLOCATE strategy.
 INSERT INTO "ext"."retention_policy"
+  (strategy, table_schema, table_name, partition_column, retention_period, reallocation_attribute_name, reallocation_attribute_value)
 VALUES
-  ('doc', 'raw_metrics', 'ts_day', 60, 'storage', 'cold', NULL, 'reallocate');
+  ('reallocate', 'doc', 'raw_metrics', 'ts_day', 60, 'storage', 'cold');
 ```
 
+This retention policy implements the following directive.
+> **Reallocate** data from the `"doc"."raw_metrics"` table to nodes tagged with
+> `node.attr.storage=cold`, on partitions defined by column `ts_day`, when it is
+> older than **60** days.
```

Comment:
After your approval at https://github.com/crate-workbench/cratedb-retention/pull/17#issuecomment-1612474406, this has been added here now.